### PR TITLE
Bug 1635897 - use repos.getCommit for getting releases' sha

### PR DESCRIPTION
**Pull request by djmitche**
Bugzilla Bug: [1635897](https://bugzilla.mozilla.org/show_bug.cgi?id=1635897)

### CODE REVIEW

**Comment by djmitche** [comment]
I'm not particularly happy with the testing situation here but it at least covers the affected code.

File: services/github/test/handler_test.js [hunk 1]
```diff
@@ -278,6 +278,12 @@ helper.secrets.mockSuite(testing.suiteName(), ['db'], function(mock, skipping) {
         'event.base.sha': base || '2bad4edf90e7d4fb4643456a4df333da348bbed4',
         'event.head.user.id': 190790,
       };
+      if (eventType === 'release') {
+        delete details['event.head.sha'];
+        delete details['event.head.ref'];
+        details['event.version'] = 'v1.2.3';
+        // NOTE: this is still not an accurate `details` for relases..
```
Comment (imbstack) [inline services/github/test/handler_test.js:285]: maybe a bit more info here for future spelunkers?

**Comment by djmitche** [comment]
> was `getRef` mocked before? we could remove it now if it was

Nope - this was untested.

**Comment by djmitche** [comment]
I updated the comment

**Pull request merged**